webapp/channels/src/utils/utils.tsx

```diff
@@ -415,6 +415,9 @@ export function applyTheme(theme: Theme) {
         changeCss('.app__body .emoji-picker .nav-tabs li a', 'fill:' + theme.centerChannelColor);
         changeCss('.app__body .post .post-collapse__show-more-button', `border-color:${changeOpacity(theme.centerChannelColor, 0.1)}`);
         changeCss('.app__body .post .post-collapse__show-more-line', `background-color:${changeOpacity(theme.centerChannelColor, 0.1)}`);
+        if (!UserAgent.isFirefox() && !UserAgent.isInternetExplorer() && !UserAgent.isEdge()) {
```

Comment:
@hmhealey A bit rusty on CSS so I was wondering... instead of this, shouldn't we use `var(--center-channel-color-rgb)` in `webapp/channels/src/sass/components/_scrollbar.scss`? Or doing it this way is in any way better?

Reply:
The other way is actually better. We want to get rid of `applyTheme` eventually since it's harder to use and has worse performance